index.js

```diff
@@ -193,7 +197,23 @@ module.exports = (config) => {
 
 					const res = await testrail.addPlanEntry(config.plan.existingPlanId, data);
 					runId = config.runId ? config.runId : res.runs[0].id;
-				} else {
+				} else if (config.plan) {
```

Comment:
I have a feeling that we can simplify the check here and we could avoid the code lines.

```
if (config.plan) {
    if (config.plan.existingPlanId) {

         let data = {
						suite_id: suiteId,
						name: runName,
						include_all: !config.plan.onlyCaseIds,
						config_ids,
						runs: [{
							include_all: false,
							case_ids: ids,
							config_ids
						}]
					};
 
      if (config.plan.onlyCaseIds) {
           data = { ...data, case_ids: ids  }
      }
      ....
```